tox.ini

```
@@ -27,6 +27,7 @@ basepython = python3.7
 deps = pip~=20.3
 whitelist_externals = bash
 setenv =
+    DUMMY_RUN = True
```

It's quite ugly to have this strewn in all the tutorials, I think. Users shouldn't have to know or care about our continuous integration system to read the tutorials!

At the very least we should call it something less informal: maybe

```suggestion
    PARTIAL_RUN = True
```

And then use it less conspicuously (and with no mention of testing):

```python
FULL_RUN = not os.environ.get("PARTIAL_RUN")  # full execution or quick partial run?

num_query_points = 10 if FULL_RUN else 2
```